main.go

```diff
@@ -143,11 +157,77 @@ func watchGGST(noClose bool, ctx context.Context) {
 	}
 }
 
+func autoUpdate() {
+
+	url := GithubDownloadURL
+	if UngaBungaMode == "" {
+		url = GithubDownloadURL + UpdateName
+	} else {
+		url = GithubDownloadURL + UpdateNameUngaBunga
+	}
+
+	client := http.Client{
+		Timeout: UpdateTimeout,
+	}
+	resp, err := client.Get(GithubReleasesURL)
+
+	if err != nil {
+		fmt.Println("Couldn't get latest release version number")
+		return
+	}
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		fmt.Println("Couldn't read latest release version")
+		return
+	}
+
+	release := &Release{}
+	json.Unmarshal(body, release)
+
+	if release.TagName != Version && Version != "" {
+		exePath, err := os.Executable()
+		if err != nil {
+			fmt.Println("Failed to get executable path")
+			return
+		}
+
+		fmt.Println("New version released, downloading...")
+		resp, err := http.Get(url)
+		if err != nil {
+			fmt.Println("Failed to download new version")
+			return
+		}
+		defer resp.Body.Close()
+
+		err = update.Apply(resp.Body, update.Options{})
+
+		if err != nil {
+			fmt.Println("Failed to update totsugeki", err)
```

Comment:
Put a divider like a `:` after `Failed to update totsugeki` so it's obvious where the error message begins.